src/game/console_cmd.c

```diff
@@ -236,6 +237,27 @@ static bool Console_Cmd_Speed(const char *const args)
     return false;
 }
 
+static bool Console_Cmd_VSync(const char *const args)
+{
+    if (String_Equivalent(args, "off")) {
+        g_Config.rendering.enable_vsync = false;
+        Config_Write();
+        Output_ApplyRenderSettings();
```

Comment:
Need to include `output.h` because you use `Output_ApplyRenderSettings`

Reply:
Thank you for spotting this; I have added the include file.